mxd/docs/File Transfer Azure to Azure.md

```diff
@@ -0,0 +1,416 @@
+| Azure Container                 | alice-container         | bob-container         | Azure Container to store files                                                                                                                                                                                                                                                                                                                 |
+| Sample File                     | alice-test-document.txt | bob-test-document.txt | A sample file already uploaded                                                                                                                                                                                                                                                                                                                 |
+
+## 3. Connect to Azurite
+To access azurite locally we need to expose the port outside our kubernetes cluster.
+```shell
+kubectl port-forward service/azurite 10000:10000
+```
+
+## 4. Upload file in Azurite
+> You can skip this step entirely if you want to use existing uploaded files.
+
+We can upload files either via Azure Storage Explorer or Azure CLI.
+
+### 4.1 Via Azure Storage Explorer
+Connect to Azurite using following details.  
+- Account Name: `aliceazureaccount`
+- Account Key: `YWxpY2VhenVyZWtleQ==`
+- Blobs port: `10000`
+
+Click on the container `alice-container` from the tree hierarchy and upload files using `Upload` button.
+
+### 4.2 using Azure CLI
+We can upload file using Azure CLI as well.
+```shell
+ export ALICE_CONN_STR='AccountName=aliceazureaccount;AccountKey=YWxpY2VhenVyZWtleQ==;DefaultEndpointsProtocol=http;BlobEndpoint=http://127.0.0.1:10000/aliceazureaccount'
+ az storage blob upload --container-name alice-container --file /path/to/file --connection-string $ALICE_CONN_STR
+```
+
+To create another container, we can run below command.
+```shell
+az storage container create --name another-alice-container --connection-string $ALICE_CONN_STR
+```
+
+
+## 5. Create Azure Asset
+
+While creating asset, we need to provide the file name, container where it is stored and azure credential to access the file.
+> Please note, a reference key (against which the real key is stored in participant's vault) in the field `dataAddress#keyName` has been passed. 
+```shell
+curl --location 'http://localhost/alice/management/v3/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {},
+  "properties": {
+    "id": "10",
+    "name": "alice-test-document",
+    "description": "Product EDC Demo Azure Asset",
+    "contenttype": "text/plain",
+    "version": "1.0"
+  },
+  "dataAddress": {
+    "@type": "DataAddress",
+    "type": "AzureStorage",
+    "account": "aliceazureaccount",
+    "container": "alice-container",
+    "blobName": "alice-test-document.txt",
+    "keyName": "aliceazureaccount-key"
+  }
+}'
+```
+
+## 6. Create Policy
```

Comment:
once again, we should have distinct policies for access and contract policy. even though it may not be technically necessary, we should explain the difference, and highlight, that they are in fact two distinct policies.<br/>
We _will_ get weird questions about this otherwise.